commons/connector/fitbit/fitbit_food_log.avsc

```diff
@@ -0,0 +1,108 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitFoodLog",
+  "doc": "A summary of the user's food log entry for a given day.",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)."
+    },
+    {
+      "name": "isFavorite",
+      "type": "boolean",
+      "doc": "Whether the food is the favorite one."
+    },
+    {
+      "name": "logId",
+      "type": "long",
+      "doc": "The id of the logged food."
+    },
+    {
+      "name": "accessLevel",
+      "type": "string",
+      "doc": "The access level of the food."
+    },
+    {
+      "name": "amount",
+      "type": "int",
+      "doc": "The amount of the food."
+    },
+    {
+      "name": "brand",
+      "type": "string",
+      "doc": "The brand of the food."
+    },
+    {
+      "name": "foodId",
+      "type": "long",
+      "doc": "The id of the food."
```

Comment:
Can you please document where this food ID comes from? For example, is it from a list or a library or repository? What is the URL for that?

Reply:
It could be from the Fitbit food database or a private food the authorized user has entered. https://dev.fitbit.com/build/reference/web-api/nutrition/get-food/